rblob/blob_test.go

```
@@ -151,6 +153,69 @@ func TestWaitForMore(t *testing.T) {
+			wantIDs: []int64{2},
+			opts:    []rblob.Option{rblob.WithBackoff(time.Millisecond)},
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			dir := tc.setup(t)
+			opts := append([]rblob.Option{rblob.WithKeyFilter(tc.filter)}, tc.opts...)
+			bucket, err := rblob.OpenBucket(context.Background(), "", "file:///"+dir, opts...)
+			require.NoError(t, err)
+			defer bucket.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `bucket.Close` is not checked

(errcheck)